regression/regression_param.sh

```diff
@@ -300,10 +300,10 @@ if [[ "$machine" = "Hera" ]]; then
    export APRUN="srun"
 elif [[ "$machine" = "Orion" ]]; then
    export OMP_STACKSIZE=2048M
-   export APRUN="srun -n \$ntasks --cpus-per-task=\$threads"
+   export APRUN="srun -n \$ntasks --mem=0  --cpus-per-task=\$threads"
 elif [[ "$machine" = "Hercules" ]]; then
    export OMP_STACKSIZE=2048M
-   export APRUN="srun -n \$ntasks --cpus-per-task=\$threads"
+   export APRUN="srun  -n \$ntasks --mem=0  --cpus-per-task=\$threads"
```

Comment:
Based on the document shared by Raghu, `--cpus-per-task` should probably be removed from these srun calls.  This is true for Jet as well.

Reply:
@DavidHuber-NOAA  .  That is a little intricate  part for me. Quote from Raghue Reddy: 
```

In your example, your tasks don't get to use all the memory if you are not using all the cores.  It is for those instances that having:

APRUN='srun -n $ntasks --mem=0 --cpus-per-task=$threads'

is needed and each task will be able to use all the available memory.
```.       So, for my understanding, using --mem=0 in the srun command line could let all available memories are used.    Since cpus-per-task is not used in the sbatch pararmeter setup,  that is why it is used in the command line.   That will be great if further comfirmation/clarification  would be made.

Reply:
I take it back.  It looks like `--cpus-per-task` should stay.
Referring to
https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php?title=Running_and_Monitoring_Jobs#Specifying_a_processor_layout_for_your_job_(uniform_layout_example)
and
https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php?title=Running_and_Monitoring_Jobs#Setting_the_--cpus-per-task_parameter,
the `--cpus-per-task` parameter is required when submitting `OpenMP` threaded jobs, so we do need it.  However, by using it, `--exclusive` is ignored.  Instead, to state that we want to use all available memory on the node, `--mem=0` should be specified, which you have done.  So all we need to add for Jet is `--mem=0`.  It seems `--exclusive` should perhaps be removed since it is ignored, but I am OK with it staying as long as `--mem=0` remains.

Reply:
@TingLei-daprediction After a conversation with Raghu, I see now that my interpretation of the `--exclusive` and `--cpus-per-task` flags was incorrect.  All three flags, `--cpus-per-task`, `--exclusive`, and `--mem=0` should be used.  So the changes you have made here are correct.  Sorry for the confusion.
@RussTreadon-NOAA I don't have permission to resolve this conversation.  Could you do that for me?